addversion.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I'd rather duplicate this code in `build.bash` and `test.bash` than have another script in the directory.